### ISSUE TITLE
Empty @scalarList returned if parent type uses integer @id

### ISSUE THREAD

**Issue by danielhopp** [prisma/prisma#4421]
**Describe the bug**
Mutations and queries return empty `@scalarList` if parent type uses integer `@id`.

**To Reproduce**
1. Deploy this `datamodel.prisma`:
```graphql
enum Option {
  A
  B
  C
}

type Something {
  id: Int! @id
  options: [Option] @scalarList(strategy:RELATION)
}
```
2. Create parent with related nodes:
```graphql
mutation {
  createSomething(data: { options: { set: [A, C] } }) {
    id
    options
  }
}
```
Result:
```javascript
{
  "data": {
    "createSomething": {
      "id": 1,
      "options": []
    }
  }
}
```

**Expected behavior**
Result:
```javascript
{
  "data": {
    "createSomething": {
      "id": 1,
      "options": ["A", "C"]
    }
  }
}
```

**Versions**
- Connector: `MySQL`, `Postgres`
- Prisma Server: `1.31.0`, `1.32.0-beta`
- `prisma` CLI: `prisma/1.31.0`, `prisma/1.31.1`, `prisma/1.32.0-beta` `(darwin-x64) node-v11.12.0`
- OS: `macOS Mojave v10.14.4`

**Additional context**
- Problem is specific to integer IDs (`id: Int! @id`), issue does not occur when using random IDs (`id: ID! @id`)
- Mutation *actually creates* scalarList nodes in database

**Comment by bkstorm** [comment]
I have the same problem.

**Comment by ZenSoftware** [comment]
I am facing this issue as well.  This occurs with a scalar list of enums also.

**Comment by pantharshit00** [comment]
I can confirm this as well.
![image](https://user-images.githubusercontent.com/22195362/56473576-df87e380-648a-11e9-8956-465ce82a9cbf.png)

**Comment by ZenSoftware** [comment]
@do4gr Unfortunately this bug is still occurring for me.  It occurs specifically with a type that uses an **Int!** for the id field.  Scalar lists on objects that use an **ID!** type for their id field seem to work fine.  Doing a raw SQL query on the relational table does show that the data is being stored and related correctly.  Though prisma queries always return an empty array.  Would you be so kind as to re-opening this issue?

## Docker images
prismagraphql/prisma:1.32.2
postgres:10.3-alpine

## npm libraries
prisma@1.32.2
prisma-binding@2.3.10

## System
node v10.15.3
Windows 10

## datamodel.prisma
```graphql
type FillBlank {
  id: Int! @id
  options: [String] @scalarList(strategy: RELATION)
}
```

**Comment by ZenSoftware** [comment]
@danielhopp Just curious if this issue has been resolved for you?

**Comment by danielhopp** [comment]
@ZenSoftware I have not tested with current master. @do4gr's recent changes concerning this issue have not been released yet.

https://github.com/prisma/prisma/compare/1.32.2..8dfb27dc84

**Comment by ZenSoftware** [comment]
My mistake, I thought that this change has been released in 1.32.

**Comment by Diablow** [comment]
Same thing is happening with UUID ids. Switching to ID works.
